pkgs/by-name/cr/crates-io-cli/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "crates-io-cli";
+  version = "4.0.2";
+
+  src = fetchFromGitHub {
+    owner = "Byron";
+    repo = "crates-io-cli";
+    rev = "v${version}";
+    hash = "sha256-28tIN8iopTQzT1RooMiB4UUxk6bHbD9V7t/yecEkLpM=";
+  };
+
+  cargoHash = "sha256-d2UKGI5CCN76KNAulGkkpw6aZTnEkCbEc1T3uPDmxC0=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
```

Without `curl` building the package failed with `: ld: library not found for -lcurl` on `aarch64-darwin`.
Is it a darwin specific dependency?
```suggestion
  buildInputs = [
  	openssl
  	curl
  ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [
  	Security
  	SystemConfiguration
  ]);
```

---

Seems some crates are making TLS connections by curl instead of openssl on darwin.